staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -375,19 +375,18 @@ func (d *DiscoveryClient) SwaggerSchema(version schema.GroupVersion) (*swagger.A
 	return &schema, nil
 }
 
-// OpenAPISchema fetches the open api schema using a rest client and parses the json.
-// Warning: this is very expensive (~1.2s)
-func (d *DiscoveryClient) OpenAPISchema() (*spec.Swagger, error) {
-	data, err := d.restClient.Get().AbsPath("/swagger.json").Do().Raw()
+// OpenAPISchema fetches the open api schema using a rest client and parses the proto.
+func (d *DiscoveryClient) OpenAPISchema() (*openapi_v2.Document, error) {
+	data, err := d.restClient.Get().AbsPath("/swagger-2.0.0.pb-v1").Do().Raw()
```

Comment:
> In other words, I don't think kubectl of any version should ever use openapi with a pre-1.7 server version.

Given code to go from spec.Schema -> openapi_v2.Document, it seems like you could hit the old endpoint, read into spec.Schema (slow as it is), then convert that to the openapi_v2.Document to return.

I'm fine telling people they shouldn't use the feature against older servers.  I'm less fine straight up breaking them.  With `--validate=true` being the default, that's what would happen with a 1.8 kubectl and 1.6 server (which a client may not be able to update), right?  Openapi for validation is a goal.